metpy/plots/tests/test_skewt.py

```diff
@@ -175,3 +175,23 @@ def test_skewt_barb_color():
     skew.plot_barbs(p, u, u, c=u)
 
     return fig
+
+
+@pytest.mark.mpl_image_compare(tolerance=0, remove_text=True)
+def test_hodograph_plot_layer():
+    """Test hodograph colored layers."""
+    u = np.arange(5., 65., 5) * units('knot')
```

Comment:
Decimals after values here and in the test data shouldn't be necessary as we don't need to force them to be floats.